This patches add compatibility for the forthcoming RobotFramework 5.

Taken from: https://github.com/robotframework/SSHLibrary/pull/403.
diff --git a/src/SSHLibrary/pythonforward.py b/src/SSHLibrary/pythonforward.py
index 8b85997b7..607985cf4 100644
--- a/src/SSHLibrary/pythonforward.py
+++ b/src/SSHLibrary/pythonforward.py
@@ -1,9 +1,9 @@
 import select
 import socket
 import threading
-from robot.utils import platform
+from robot.utils import PY2, WINDOWS
 from .logger import logger
-if platform.PY2 and platform.WINDOWS:
+if PY2 and WINDOWS:
     import win_inet_pton
 try:
     import SocketServer
